osu.Game/Online/RealtimeMultiplayer/InvalidStateChange.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+
+namespace osu.Game.Online.RealtimeMultiplayer
+{
+    public class InvalidStateChange : Exception
```

Comment:
Exception class naming in this PR is sort of half and half - there are:

* `InvalidStateChange`
* `InvalidStateException`
* `NotHostException`
* `NotJoinedRoomException`
* `UserAlreadyInMultiplayerRoom`

Would probably unify by doing `InvalidStateChange` -> `InvalidStateChangeException` and `UserAlreadyInMultiplayerRoom` -> `AlreadyInRoomException`.